presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1468,6 +1469,20 @@ public HiveClientConfig setPushdownFilterEnabled(boolean pushdownFilterEnabled)
         return this;
     }
 
+    @NotNull
+    public boolean isParquetPushdownFilterEnabled()
+    {
+        return parquetPushdownFilterEnabled;
+    }
+
+    @Config("hive.parquet.pushdown-filter-enabled")
+    @ConfigDescription("Experimental: enable complex filter pushdown for Aria Parquet")
```

Comment:
Let's remove "Aria" and say "for Parquet". Aria is a project name and we should avoid using it in the code.